ft-fine-tune-bert/fine-tune-unittests.py

```diff
@@ -127,110 +69,27 @@ def test_return_type(self):
-        training_args = TrainingArguments(
-            output_dir=output_model_path+model_name,
-            overwrite_output_dir=True,
-            do_train=True,
-            do_eval=True,
-            logging_steps=500,
-            per_device_train_batch_size=4,
-            num_train_epochs=1,
-            save_total_limit=1,
-            save_steps=1000,
-        )
-
-        # Setup logging
-        logger = logging.getLogger(__name__)
-
-        # Setup logging
-        logging.basicConfig(
-            format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
-            datefmt="%m/%d/%Y %H:%M:%S",
-            level=logging.INFO if training_args.local_rank in [-1, 0] else logging.WARN,
-        )
-        logger.warning(
-            "Process rank: %s, device: %s, n_gpu: %s, distributed training: %s, 16-bits training: %s",
-            training_args.local_rank,
-            training_args.device,
-            training_args.n_gpu,
-            bool(training_args.local_rank != -1),
-            training_args.fp16,
-        )
-        logger.info("Training/evaluation parameters %s", training_args)
-
-        config = AutoConfig.from_pretrained(model_name, cache_dir=model_args.cache_dir)
-        tokenizer = AutoTokenizer.from_pretrained(model_name, cache_dir=model_args.cache_dir)
-        model = BertLMHeadModel.from_pretrained(
-            model_name,
-            from_tf=bool(".ckpt" in model_name),
-            config=config,
-            cache_dir=model_args.cache_dir,
-        )
-        special_tokens_dict = {
-            "bos_token": "<BOS>",
-            "eos_token": "<EOS>",
-            "pad_token": "<PAD>",
-            "additional_special_tokens": [
-                "<superhero>",
-                "<action>",
-                "<drama>",
-                "<thriller>",
-                "<horror>",
-                "<sci_fi>",
-            ],
-        }
-        num_added_toks = tokenizer.add_special_tokens(special_tokens_dict)
-        model.resize_token_embeddings(len(tokenizer))
-        train_dataset = get_dataset(data_args, tokenizer=tokenizer)
-        eval_dataset = get_dataset(data_args, tokenizer=tokenizer, evaluate=True)
-        data_collator = DataCollatorForLanguageModeling(
-            tokenizer=tokenizer,
-            mlm=data_args.mlm,
-        )
-        # Initialize our Trainer
-        trainer = Trainer(
-            model=model,
-            args=training_args,
-            data_collator=data_collator,
-            train_dataset=train_dataset,
-            eval_dataset=eval_dataset,
-        )
-
-        # Train
-        if training_args.do_train:
-            trainer.train()
-
-        # Evaluation
-        if training_args.do_eval:
-            logger.info("*** Evaluate ***")
-            eval_output = trainer.evaluate()
-            perplexity = math.exp(eval_output["eval_loss"])
+        bert_target = 220
+        
+        cmd = "python -u ./finetune.py --model_name bert-base-uncased "
```

Comment:
fix the path per comment above.  

Also recommended to not hard code these parameters.  In addition try to pass the training data files  and not assume the default?